Added homework_12_211222

kate-lvlup opened the pull request:
Hi Dima,
Hope you have electricity!
Could you check my homework12 (GameApp and Palindrome)?

Thanks in advance!

YeaaaaH commented on homework_12_211222/src/Palindrome.java:
```diff
@@ -0,0 +1,131 @@
+        String input = scanner.nextLine();
+        char[] charArray = input.toCharArray();
+        Arrays.sort(charArray);
+        for (int i = 0; i < charArray.length; i++) {
```
this loop is not needed

kate-lvlup replied:
fixed

YeaaaaH commented on homework_12_211222/src/Palindrome.java:
```diff
@@ -0,0 +1,131 @@
+            System.out.println(charArray[i]);
+        }
+        String s = String.valueOf(charArray);
+        for (int i = 0; i < charArray.length; i += (s.lastIndexOf(charArray[i]) - s.indexOf(charArray[i]) + 1)) {
```
this loop is not needed

kate-lvlup replied:
I left the output as it is, so that this task is an example for future tasks.

YeaaaaH commented on homework_12_211222/src/Palindrome.java:
```diff
@@ -0,0 +1,131 @@
+            }
+            res = s.lastIndexOf(charArray[i]) - s.indexOf(charArray[i]) + 1;
+        }
+        System.out.println(max);
```
this output is not needed

kate-lvlup replied:
I left the output as it is, so that this task is an example for future tasks.

kate-lvlup commented:
Added a commit with fixes and also:
- changed the name of the Polyndrom class to MethodCheck;
- changed the name of the count() method to printScore();
- made the methods randomChoice(), restartGame(), printScore() non-static;
- simplified the code in the GameApp class ("Draw between players!")

YeaaaaH reviewed (approved):
nice
